web/src/components/layout/nav/contribute-menu/contribute-menu.css

```diff
@@ -188,6 +189,15 @@
             width: auto;
         }
 
+        p.coming-soon-text {
```

Comment:
This should not be in this PR I assume?

Reply:
Answered on Slack but also mentioning here for publicity:

>There's a different type of coming soon that we have - for locales that are not yet contributable we show coming soon for the speak and listen links till they are contributable
When they are contributable, the normal text shows